mkosi/config.py

```diff
@@ -858,6 +858,8 @@ def config_parse_dict(value: Optional[str], old: Optional[dict[str, Any]]) -> Op
             else:
                 die(f"{p} does not exist")
 
+            return new
```

Comment:
We also need an additional check that `value` is not an empty string to make sure we don't enter this `if` if we get an empty string.

Reply:
Nvm don't bother with this I added it myself as part of #2620